### ISSUE TITLE
[meta] Get higher score on Observatory

### ISSUE THREAD

**Issue by freddyb** [mozilla/standup#224]
https://observatory.mozilla.org/analyze.html?host=standu.ps

- [ ] CSP #220
- [ ] HSTS #221 
- [ ] HTTP to HTTPS redirect #222
- [ ] Security headers (XFO et al.)  #223

**Comment by freddyb** [comment]
To be fair, this isn't about the Observatory score, but rather about best practices in web security :)

**Comment by pmac** [comment]
Most of this is predicated on getting the cert, which according to [the bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1300770#c1) is in progress.

**Comment by pmac** [comment]
Which we're tracking in #168.

**Comment by freddyb** [comment]
Yes. To be clear, this isn't me complaining. I'm happy to help do those things via pull requests. :)

**Comment by pmac** [comment]
We're up to an `A`! https://observatory.mozilla.org/analyze.html?host=www.standu.ps

**Comment by pmac** [comment]
I'll be bumping the HSTS header value to 1 year later this week. Just want to give it some time before locking us in to discover any issues.
